Formula/netdata.rb

```diff
@@ -0,0 +1,64 @@
+class Netdata < Formula
+  desc "Get control of your servers. Simple. Effective. Awesome!"
```

Comment:
We'll need a description that is a bit more description-y and a bit less marketing-y.